src/api/commands/listingitem/ListingItemFlagCommand.ts

```diff
@@ -66,17 +66,6 @@ export class ListingItemFlagCommand extends BaseCommand implements RpcCommandInt
         const optionsList: string[] = [ItemVote.KEEP, ItemVote.REMOVE];
         const proposalTitle = listingItemHash;
 
-        // TODO: refactor these to startTime and endTime
-        // TODO: When we're expiring by time not block make this listingItem.ExpiryTime();
```

Comment:
blockStart -> timeStart, rather than startTime, since startTime is allready used elsewhere and it'll waste time when doing a search.
blockEnd -> timeEnd, same thing.

timeStart = now();
timeEnd = listingItem.expiredAt;
daysRetention = calculateDaysRetention(now(), timeEnd);

timeStart, timeEnd should be in the same format as postedAt and the other fields in SmsgMessage. Ignore createdAt and updatedAt which are automatically generated fields by Bookshelf.

help() is missing reason.

Reply:
...[3]: expiryTime (from listingitem, set in validate) 
^^ incorrect, should be expiredAt, either remove the param and fetch the listingItem again in here or set the third param to listingItem.expiredAt